lambdas/pkgpush/index.py

```diff
@@ -29,6 +30,11 @@
 PROMOTE_PKG_MAX_FILES = int(os.environ['PROMOTE_PKG_MAX_FILES'])
 PKG_FROM_FOLDER_MAX_PKG_SIZE = int(os.environ['PKG_FROM_FOLDER_MAX_PKG_SIZE'])
 PKG_FROM_FOLDER_MAX_FILES = int(os.environ['PKG_FROM_FOLDER_MAX_FILES'])
+S3_HASH_LAMBDA = os.environ['S3_HASH_LAMBDA']
+S3_HASH_LAMBDA_CONCURRENCY = int(os.environ['S3_HASH_LAMBDA_CONCURRENCY'])
```

Comment:
we should call this something else. because it's not the same as lambda reserved concurrency. and that also makes me wonder what happens if this number is greater than reserved concurrency and how we keep those numbers in sync?